sdk/include/opentelemetry/sdk/logs/logger_config.h

```diff
@@ -0,0 +1,60 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace logs
+{
+/**
+ * LoggerConfig defines various configurable aspects of a Logger's behavior.
+ * This class should not be used directly to configure a Logger's behavior, instead a
+ * ScopeConfigurator should be used to compute the desired LoggerConfig which can then be used to
+ * configure a Logger.
+ */
+class LoggerConfig
+{
+public:
+  bool operator==(const LoggerConfig &other) const noexcept;
+
+  /**
+   * Returns if the Logger is enabled or disabled. Loggers are enabled by default.
+   * @return a boolean indicating if the Logger is enabled. Defaults to true.
+   */
+  bool IsEnabled() const noexcept;
+
+  /**
+   * Returns a LoggerConfig that represents an enabled Logger.
+   * @return a static constant LoggerConfig that represents an enabled logger.
+   */
+  static LoggerConfig Enabled();
```

Comment:
Because this class can be used as a parameter of factory, which is public and exported. I think we should use `OPENTELEMETRY_EXPORT` to export all public static functions and default constructor, destructor and assignment operators.

Reply:
The comments around this Macro do not provide the details about its usages. What does "exporting" public static functions mean in this context ?

Reply:
Thank you for the explanation, I have made this change.

Based on this, I think similar change should be made for [meter_config](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/sdk/src/metrics/meter_config.cc) and [tracer_config](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/sdk/src/trace/tracer_config.cc) ?

I think private static  variables could be applied there as well.

Reply:
Created issue to track this work for tracer and meter config. 

https://github.com/open-telemetry/opentelemetry-cpp/issues/3302